public/app/plugins/datasource/prometheus/language_provider.ts

```diff
@@ -471,6 +471,10 @@ export default class PromQlLanguageProvider extends LanguageProvider {
     }
   }
 
+  /**
+   * @todo cache
```

Comment:
This wasn't currently cached, do we want to attempt to cache these values like we do with the series result queries?